packages/feature-toggle/README.md

```diff
@@ -0,0 +1,93 @@
+# @sap-ux/feature-toggle
+
+This module can be used for accessing and managing feature toggles in Fiori tools.
```

Comment:
Is it Fiori tools specific?

Reply:
I would have to say yes, based on packages/feature-toggle/src/constants.ts
It is only Fiori tools toggles which are in this module and I presume going forward it would remain that way?